backend/runners/load_representative_points.py

```diff
@@ -47,27 +47,33 @@ def _main(kwargs):
 def _insert_service_areas(json_features):
     """Insert service areas into the database from a GeoJSON file."""
     data = _get_all_service_areas(json_features)
-    methods.core_insert(
-        engine=connect.create_db_engine(),
-        sql_class=service_area.ServiceArea,
-        data=data,
-        return_insert_ids=False,
-        unique=False
-    )
-    return data
+    try:
+        methods.core_insert(
+            engine=connect.create_db_engine(),
+            sql_class=service_area.ServiceArea,
+            data=data,
+            return_insert_ids=False,
+            unique=False
+        )
+        return data
+    except Exception as e:
+        print('Error inserting service areas: {}'.format(e))
 
 
 def _insert_representative_population_points(json_features):
     """Insert representative points into the database from a GeoJSON file."""
     data = [_transform_single_point(point) for point in json_features]
-    methods.core_insert(
-        engine=connect.create_db_engine(),
-        sql_class=representative_point.RepresentativePoint,
-        data=data,
-        return_insert_ids=False,
-        unique=False
-    )
-    return data
+    try:
+        methods.core_insert(
+            engine=connect.create_db_engine(),
+            sql_class=representative_point.RepresentativePoint,
+            data=data,
+            return_insert_ids=False,
+            unique=False
+        )
+        return data
+    except Exception as e:
+        print('Error inserting representative points')
```

Comment:
Should we use logging here as well?

Reply:
My rationale was that these scripts aren't really part of the application as such, so shouldn't necessarily use the application logging. Can easily change though.

Reply:
Yeah that's true. But we are using the standard logging and not the app logging atm